include/cbl/CBLDatabase.h

```diff
@@ -51,14 +50,16 @@ typedef struct CBLEncryptionKey {
     CBLEncryptionAlgorithm algorithm;       ///< Encryption algorithm
     uint8_t bytes[32];                      ///< Raw key data
 } CBLEncryptionKey;
+#endif
 
 /** Database configuration options. */
 typedef struct {
-    FLString directory;                     ///< The parent directory of the database
-    CBLEncryptionKey* encryptionKey;        ///< The database's encryption key (if any)
+    FLString directory;                         ///< The parent directory of the database
+#ifdef COUCHBASE_ENTERPRISE
+    const CBLEncryptionKey* encryptionKey;      ///< The database's encryption key (if any)
```

Comment:
There doesn't seem to be a reason anymore to have this be a pointer. Why not just put the key struct itself in the configuration? If you zero-initialize the config (`CBLDatabaseConfig config {};`) the key will be set to none.